TopAnalysis/src/TTbarDoubleLeptonCppWorker.cc

```diff
@@ -262,6 +262,27 @@ bool TTbarDoubleLeptonCppWorker::analyze() {
     out_Jets_CSVv2[k] = in_Jets_CSVv2->At(kk);
   }
 
+  // Get CutStep
+  if ( actualMode == MODE::MuMu ) {
+    if ( out_Z_charge == 0 and out_Lepton1_p4[0] > 25 and out_Lepton2_p4[0] > 20 ) out_nCutStep = 1;
+      if ( out_Z_p4[0] < 75 or out_Z_p4[0] > 105 ) out_nCutStep = 2;
+        if ( out_MET_pt > 40 ) out_nCutStep = 3;
+          if ( out_nGoodJets > 3 ) out_nCutStep = 4;
+            if ( out_nGoodBjets > 1 ) out_nCutStep = 5;
+  }
+  else if ( actualMode == MODE::ElEl ) {
+   if ( out_Z_charge == 0 and out_Lepton1_p4[0] > 25 and out_Lepton2_p4[0] > 20 ) out_nCutStep = 1;
+      if ( out_Z_p4[0] < 75 or out_Z_p4[0] > 105 ) out_nCutStep = 2;
+        if ( out_MET_pt > 40 ) out_nCutStep = 3;
+          if ( out_nGoodJets > 3 ) out_nCutStep = 4;
+            if ( out_nGoodBjets > 1 ) out_nCutStep = 5;
+  }
+  else if ( actualMode == MODE::MuEl ) {
+   if ( out_Z_charge == 0 and out_Lepton1_p4[0] > 25 and out_Lepton2_p4[0] > 20 ) out_nCutStep = 3;
+     if ( out_nGoodJets > 3 ) out_nCutStep = 4;
+       if ( out_nGoodBjets > 1 ) out_nCutStep = 5;
+  }
+
```

Comment:
Wrap with braces {}. This code will reassign cutflow to step 5 if Z_charge !=0 and out_nGoodBJetts == 2.